.github/workflows/at_client.yml

```diff
@@ -26,6 +26,9 @@ jobs:
           distribution: 'adopt'
           cache: maven
           server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
-
-      - name: Test and Build with Maven
-        run: mvn --batch-mode --update-snapshots --no-transfer-progress -Dgpg.skip=true clean install
+      - name: Build clean and run unit tests
+        run: mvn --batch-mode clean test -DskipIntegrationTests=true
```

Comment:
`-Dgpg.skip=true` add this flag too - I am scared `test` will run the signing plugin

Reply:
Done